### ISSUE TITLE
Charge Status doesn't work on Model 3

### ISSUE THREAD

**Issue by jtmcdole** [lyceel/tesla_dart#3]
0 / 0 :(
```
Charge State:
  Battery:         0.0/0.0 kWh (59%)
                   ╭──────────╮
                   │█████▉    ▐
                   ╰──────────╯
```

**Comment by lyceel** [comment]
Can you run with --show-all --raw and send me the output?

**Comment by jtmcdole** [comment]
Note: sent via email. The REST api is reporting odd codes and is missing codes for the paint color matching... then I saw [this](https://teslamotorsclub.com/tmc/threads/decoding-model-3-option-codes.109297/): 

> I run the A Better Routeplanner service where we let users login via MyTesla and optionally share data with us. For fun and statistics, I just checked the option codes for all 258 Model 3s in our database. They are all the same.

**Comment by lyceel** [comment]
Should be fixed by f8c4739
